client/public/index.html

```diff
@@ -24,11 +24,14 @@
       work correctly both with client-side routing and a non-root public URL.
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
-    <title>React App</title>
+    <title>Michael Muzzy</title>
   </head>
   <body>
     <noscript>You need to enable JavaScript to run this app.</noscript>
-    <div id="root"></div>
+    
+    <div id="root">
+      <h1><a href='/'>Home</h1>
```

Comment:
It looks like this isn't being used right here, you can take this right out of your html